src/main/java/fr/cnes/sonar/report/providers/ComponentProvider.java

```diff
@@ -133,6 +140,11 @@ private double getMaxMetric(String metric){
 
 
     private boolean isCountableMetric(String metric){
+        // The metric is to be excluded
+        if(excludeMetricSet.contains(metric)){
+            return false;
```

Comment:
Could you please refactor your code so that this method only has one return statement ?